src/modules/jsUtils.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
I see, right, you have an error since you are in node context. I think that is OK to have a separate module that the server is free to uses. But I would suggest to put that module in other location, since that could be confusing. One can think that if the server can require this module, under `src/modules/` then any other server thing can be placed in here, when it is not alwasy true. The intention of the modules that are in this folder I think that it is for browser usages, by using browserify.
Maybe we can improve a little how the `utils` module is implemented, in order to separate functions that are more useful for simulation, like navigation utils for example, from functions that we can have in a kind of utility library itself. Then we can point in the browserify configuration, to load that module to.